src/components/ProposalApplication.tsx

```diff
@@ -66,7 +89,44 @@ export default function ProposalApplication({
                     label: 'Details',
                     accessor: 'details',
                     transformer: ({ row }) => (
-                      <ApplicationDetailsModal row={row} />
+                      <ApplicationDetailsModal
+                        row={row}
+                        isModalOpen={isModalOpen}
+                        setIsModalOpen={setIsModalOpen}
+                      />
+                    ),
+                  },
+                  {
+                    label: 'Action',
+                    accessor: 'action',
+                    transformer: ({ row }) => (
+                      <Button
+                        disabled={
+                          row.statusKey === 'ACCEPTED' ||
+                          acceptApplication.isLoading
+                        }
+                        onClick={async () => {
+                          await acceptApplication.mutateAsync(
+                            {
+                              proposalId: proposalDetails.id,
+                              proposalApplicationId: row.id,
+                              applicantEmail: row.email,
+                            },
+                            {
```

Comment:
@rschlaefli - refetching works (can see in networking request), but the page is not re-rendered